### PR TITLE
FEAT: lightbox dialog without background click

### PR DESCRIPTION
**Pull request by montoya332**
disabling background click is not supported in Lightbox .

### CODE REVIEW

File: src/ebay-confirm-dialog/confirm-dialog.tsx [hunk 1]
```diff
@@ -40,24 +40,28 @@ const EbayConfirmDialog: FC<Props> = ({
...
-                    aria-describedby={mainId}
-                    className="confirm-dialog__confirm"
-                >
-                    {confirmText}
-                </EbayButton>
-            </EbayDialogFooter>
+            {
+                (confirmText || rejectText) &&
+                <EbayDialogFooter>
+                    <EbayButton
```
Comment (HenriqueLimas) [inline src/ebay-confirm-dialog/confirm-dialog.tsx:46]: Should we render those button only when the text exists?

```jsx
(confirmText || rejectText) && 
<EbayDialogFooter>
  {rejectText && <EbayButton />}
  {confirmText && <EbayButton />}
</EbayDialogFooter>
```

In that way there is't a risk to render an empty button

Comment (montoya332) [reply src/ebay-confirm-dialog/confirm-dialog.tsx]: moved to button position logic

File: src/ebay-dialog-base/components/dialogBase.tsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ export const DialogBase: FC<DialogBaseProps<HTMLElement>> = ({
                 onBackgroundClick(e)
             }
         }
-        if (open) {
+        if (open && buttonPosition !== 'hidden') {
```
Comment (montoya332) [inline src/ebay-dialog-base/components/dialogBase.tsx:92]: https://github.com/eBay/ebayui-core/blob/6c22e9e3a9d5d546cbc7667ff4ee96a4186f42b4/src/components/components/ebay-dialog-base/component.js#L55

File: src/ebay-lightbox-dialog/lightbox-dialog.tsx [hunk 3]
```diff
@@ -22,10 +22,10 @@ const EbayLightboxDialog: FC<Props> = ({
     ...rest
 }) => (
     <DialogBaseWithState
+        buttonPosition="right"
```
Comment (montoya332) [inline src/ebay-lightbox-dialog/lightbox-dialog.tsx:25]: moving up to override if needed

**Pull request merged**